internal/controller/dragonfly_pod_lifecycle_controller.go

```diff
@@ -95,7 +95,7 @@ func (r *DfPodLifeCycleReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 			}
 
 			r.EventRecorder.Event(dfi.df, corev1.EventTypeNormal, "Replication", "configured replication for first time")
-		} else if dfi.df.Status.Phase == PhaseReady {
+		} else if dfi.df.Status.Phase == PhaseReady || dfi.df.Status.Phase == PhaseRollout {
```

Comment:
Following our discussion - yea I think it will be better to introduce another status field IsRollingUpdateInProgress and then this controller won't have to deal with this aspect at all, right?

Reply:
Yep, Updated!